src/lighteval/tasks/default_tasks.py

```diff
@@ -324,10 +324,14 @@
     few_shots_select=None,
     generation_size=32768,
     metric=[
-        Metrics.expr_gold_metric,
```

Comment:
This was causing redundant computation since we can get the same result from pass@1 (n=1)